### PR TITLE
Software permissions table

### PR DESCRIPTION
**Pull request by jwitz**
Resolves https://github.com/astronomer/docs/issues/3046 (pt 2)

New visual tables that show user permissions, for those that are just interested in defaults rather than customization

### CODE REVIEW

File: software/role-permission-reference.md [hunk 1]
```diff
@@ -7,9 +7,95 @@ description: A list of all default permissions for each role on Astronomer Softw
...
+| Deployment Admin permissions for all Deployments                                                                                                  | ✔️                 | ✔️                 | ✔️                |
+| View [environment variables](environment-variables.md) for any Deployment                                                                         | ✔️                 | ✔️                 | ✔️                |
+| View any setting for any Deployment in the Software UI                                                                                            | ✔️                 | ✔️                 | ✔️                |
+| View all pending user invites in the **System Admin** tab of the Software UI                                                                      | ✔️                 | ✔️                 | ✔️                |
+| View information for any pending user invite                                                                                                      | ✔️                 | ✔️                 | ✔️                |
+
```
Comment (jwitz) [inline software/role-permission-reference.md:70]:
```suggestion
```

File: software/role-permission-reference.md [hunk 2]
```diff
@@ -7,9 +7,94 @@ description: A list of all default permissions for each role on Astronomer Softw
 
 This is where you'll find information about Astronomer Software default user role permissions. To modify these default permissions, see [Customize role permissions](manage-platform-users.md#customize-role-permissions).
 
-## System roles
-
-System roles apply to all Workspaces, users, and Deployments within a single Astronomer Software installation.
+## Default role permissions tables
+
+The following tables show high level comparisons of permissions between different user roles.
```
Comment (lzdanski) [inline software/role-permission-reference.md:12]:
```suggestion
The following tables show high level comparisons of the different permitted actions between different user roles.
```

File: software/role-permission-reference.md [hunk 3]
```diff
@@ -7,9 +7,94 @@ description: A list of all default permissions for each role on Astronomer Softw
...
+| Bypass email verification for any user                                                                                                            |                   |                   | ✔️                |
+| Create, update, or delete a Workspace                                                                                                             |                   |                   | ✔️                |
+| Airflow [admin permissions](https://airflow.apache.org/docs/apache-airflow/stable/security/access-control.html#admin) for all Deployments         |                   |                   | ✔️                |
+
+## Default role permissions lists
+
+The following topics list the permission values that each role has by default as defined in the Astronomer Helm chart. You can update these permissions in your `config.yaml` file if you want to change the permissions that each role has. See [Customize role permissions](manage-platform-users.md#customize-role-permissions).
```
Comment (lzdanski) [inline software/role-permission-reference.md:95]:
```suggestion
The following sections list the permission values that each role has by default as defined in the Astronomer Helm chart. You can update these permissions in your `config.yaml` file if you want to change the permissions that each role has. See [Customize role permissions](manage-platform-users.md#customize-role-permissions). 
```

File: software/role-permission-reference.md [hunk 4]
```diff
@@ -7,9 +7,94 @@ description: A list of all default permissions for each role on Astronomer Softw
...
+| Bypass email verification for any user                                                                                                            |                   |                   | ✔️                |
+| Create, update, or delete a Workspace                                                                                                             |                   |                   | ✔️                |
+| Airflow [admin permissions](https://airflow.apache.org/docs/apache-airflow/stable/security/access-control.html#admin) for all Deployments         |                   |                   | ✔️                |
+
+## Default role permissions lists
+
+The following topics list the permission values that each role has by default as defined in the Astronomer Helm chart. You can update these permissions in your `config.yaml` file if you want to change the permissions that each role has. See [Customize role permissions](manage-platform-users.md#customize-role-permissions). 
+
+Note that these lists are also published in YAML form on the [Astronomer documentation repository](https://github.com/astronomer/docs/blob/main/software_configs/0.33/default.yaml).
```
Comment (lzdanski) [inline software/role-permission-reference.md:97]:
```suggestion
These lists are also published in YAML form in the [Astronomer documentation repository](https://github.com/astronomer/docs/blob/main/software_configs/0.33/default.yaml).
```

**Review by lzdanski — APPROVED** [review]
Minor feedback, but overall lgtm

**Pull request merged**